### PR TITLE
Add RHEL/EC2 support

### PR DESCRIPTION
**Pull request by netflash**
All tests are passed.

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -1,22 +1,40 @@
...
-  name: vagrant
-  vm_hostname: devopsbox.local
+  name: ec2
+  hostname: bastion.local
+driver_config:
+  interface: private
+  instance_type: t2.medium
+  region: us-west-2
+  subnet_id: subnet-XXXXXX
+  security_group_ids: [ "sg-XXXXXX" ]
```
Comment (toddmichael) [inline .kitchen.yml]: `-XXXXX`? Are these faux/placeholder values?

File: .kitchen.yml [hunk 2]
```diff
@@ -1,22 +1,40 @@
...
-  require_chef_omnibus: true
+  require_chef_omnibus: 12.13.37
 
-# Uncomment the following verifier to leverage Inspec instead of Busser (the
-# default verifier)
-# verifier:
-#   name: inspec
+#verifier:
+#  name: inspec
+#  sudo: true
```
Comment (toddmichael) [inline .kitchen.yml]: I've been filing Github issues to migrate to InSpec. Recommend doing that if you don't have time to migrate.

File: .kitchen.yml (hunk 1 above)
Comment (netflash) [inline .kitchen.yml]: right, that's it. #9 to cover this issue.

File: .kitchen.yml (hunk 2 above)
Comment (netflash) [inline .kitchen.yml]: #10 it is

**Pull request merged**